KIPs/kip-114.md

```diff
@@ -159,43 +159,52 @@ Therefore, in theory, no one (including the proposer of the block `N`) can predi
+In other words, `f(x, numVals, numCols)` is zero when `x >= 2*numVals` and `numVals > numCols`.
+
+#### After KIP-146
+
+After KIP-146, the proposer is randomly selected after each block, thus `f(x, numVals, numCols) = (numCols/numVals)^x`.
+
 ### Tips for application developers
 
-In Cypress, the proposer is selected in a round-robin manner; all validators get an equal chance to propose in each round. Users of this random should be aware that in the worst-case scenario where only a single validator remains benign, the colluding validators can predict the value of `mixHash` ahead by up to twice the number of colluding validators (i.e., when benign validator proposes first in the first round and last in the second round). Note that this number may differ should the proposer selection algorithm change.
+Before KIP-146, developers are advised to give a window of twice the number of validators between the commit and reveal block when using commit-reveal random schemes using `mixHash`.
 
-After the activation of [KIP-146](https://github.com/klaytn/kips/blob/kip114/KIPs/kip-146.md), the predictable window size increases.
-Define `f(x, N, C) = (C/N)^x` as the probability of `C` number of colluding validators, among `N` validators, conspiring to predict the value of `mixHash` `x` blocks in advance.
-It implies that `C` number of colluding validators must become proposers for `x` consecutive blocks.
-Here are some varying combinations of `N`, `C`, and `threshold`, which developers can refer to for choosing the security level.
+After KIP-146, there is no theoretical bound to which `mixHash` becomes unpredictable. Here are some varying combinations of `N`, `C`, and `threshold`, which developers can refer to for choosing the security level.
 
-| N   | C   | x where f(x, N, C) < 0.001 | x where f(x, N, C) < 0.0001 |
-| --- | --- | -------------------------- | --------------------------- |
-| 30  | 1   | 3                          | 3                           |
-| 30  | 21  | 20                         | 26                          |
-| 30  | 29  | 204                        | 272                         |
-| 100 | 1   | 2                          | 3                           |
-| 100 | 10  | 4                          | 5                           |
-| 100 | 50  | 10                         | 14                          |
-| 100 | 67  | 18                         | 23                          |
-| 100 | 99  | 688                        | 917                         |
+| numVals | numCols | x where f(x, numVals, numCols) < 0.001 | x where f(x, numVals, numCols) < 0.0001 |
```

Comment:
The proposed change does not appear to enhance readability. In my opinion, the previous version seems more concise.

Reply:
I initially thought so, but I wanted to make search easier to find the definition for those that didn't read the above parts. I don't have a preference on this issue. let me ask others to decide